cmd/erasure-multipart.go

```diff
@@ -836,12 +869,18 @@ func (er erasureObjects) CompleteMultipartUpload(ctx context.Context, bucket str
 // Implements S3 compatible Abort multipart API, slight difference is
 // that this is an atomic idempotent operation. Subsequent calls have
 // no affect and further requests to the same uploadID would not be honored.
-func (er erasureObjects) AbortMultipartUpload(ctx context.Context, bucket, object, uploadID string) error {
+func (er erasureObjects) AbortMultipartUpload(ctx context.Context, bucket, object, uploadID string, opts ObjectOptions) error {
```

Comment:
The documentation states that this is idempotent, however as far as I can tell the first will return OK and the following will return "not found".

Shouldn't this return no error if not found?

Reply:
No AbortMultipartUpload should return InvalidUpload it is the handler that returns the appropriate error.

Reply:
That comment should be removed